packages/devextreme/js/__internal/ui/form/form.ts

```diff
@@ -1846,8 +1880,14 @@ class Form extends Widget<FormProperties> {
       instruction: item.aiOptions?.instruction,
     }));
 
+    const clipboardText = text ?? await navigator.clipboard?.readText();
+    if (!clipboardText) {
+      this._hideLoadPanel();
```

Comment:
why do we silently bury the request? without even a warning in console? or showing something to a user?